react-backend/client/src/components/visitors/qrcode.js

```diff
@@ -2,9 +2,39 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
+import styled from 'styled-components';
 import PurposeButton from './purposeButton';
 import QRPrivacy from '../visitors/qrprivacy';
 import { Activities, Visitors } from '../../api';
+import { Heading } from '../../shared/components/text/base';
+import { FlexContainerRow } from '../../shared/components/layout/base';
+
+const StyledNav = styled.nav`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const StyledSection = styled.section`
+  margin: 3rem 0;
+  display: flex;
+  justify-content: center;
+`;
+
+const HalfFlexContainerRow = FlexContainerRow.extend`
+  width: 50%;
+  padding: 1rem;
+  flex-wrap: wrap;
+`;
+
+const SnakeContainerRow = FlexContainerRow.extend`
```

Comment:
Could you quickly explain why this is necessary?

Reply:
This is creating a div  which grabs the second child and reverses it's direction. When the divs are stacked on top of each other the elements order is concertinaed like a snake.